src/Google.Api.Gax.Grpc/UserAgentBuilder.cs

```diff
@@ -48,10 +48,10 @@ internal UserAgentBuilder AppendAssemblyVersion(string name, System.Type type)
         /// </summary>
         internal UserAgentBuilder AppendDotNetEnvironment()
 #if NETSTANDARD1_5
-            // TODO: Improve this if we can.
-            => AppendVersion("dotnet", "unknown");
+            // TODO: Improve this if we can. (At least 0.0.0 is valid semver, so should parse.)
+            => AppendVersion("gl-dotnet", "0.0.0");
```

Comment:
It was changed in https://github.com/googleapis/gax-dotnet/pull/134 to fetch the version for .NET Core.